src/pages/InitMsig.vue

```diff
@@ -0,0 +1,264 @@
+										<a-button
+											@click="item.edit = !item.edit"
+											type="link"
+											shape="circle"
+											size="small"
+											ghost
+											><div v-if="!item.edit">
+												<IconWithToolTip :data="'Edit'" :icon="'Edit'" />
+											</div>
+											<div v-else>
+												<IconWithToolTip :data="'Save'" :icon="'Save'" />
+											</div>
+										</a-button>
+										<a-button
+											@click="removeAddress(item)"
+											type="link"
+											danger
+											shape="circle"
+											size="small"
+											ghost
+										>
+											<IconWithToolTip :data="'Delete'" :icon="'Delete'" />
+										</a-button>
+									</template>
+								</a-list-item>
+							</template>
+						</a-list>
+
+						<li class="list_addresses">
+							<a-input-search
+								v-model:value="newAddress"
+								placeholder="Add address"
+								enter-button="+"
+								@search="addAddress"
+							/>
+						</li>
+						<p class="margin_top_med">
+							Threshold:
+							<a-input
+								type="number"
+								placeholder="Threshold"
+								v-model:value="threshold"
+							/>
+						</p>
+						Version:
+						<a-input
+							type="number"
+							placeholder="Version"
+							v-model:value="version"
+						/>
+					</a-card>
+				</div>
+				<a-button type="primary" @click="sign" class="margin_top_med"
+					>SIGN</a-button
+				>
+			</a-col>
+			<a-col :xs="{ span: 24 }" :lg="{ span: 11, offset: 2 }">
+				<h3>Transaction preview</h3>
+				<a-card
+					class="card"
+					title="Format Transaction"
+					:tab-list="tabList"
+					:active-tab-key="key"
+					:bordered="false"
+					@tabChange="(key) => onTabChange(key, 'key')"
+				>
+					<template #customTab="item">
+						<span>
+							{{ item.key }}
+						</span>
+					</template>
+					<a-textarea
+						class="text_area"
+						:auto-size="{ maxRows: 22 }"
+						:bordered="false"
+						v-model:value="contentList[key]"
+						:disabled="true"
+					/>
+				</a-card>
+				<MultisigParameters :inputBase64="contentList.MSG_PACK" />
+			</a-col>
+		</a-row>
+	</a-layout-content>
+</template>
+
+<script lang="ts">
+import { contentlist, listAddresses, WalletType } from "@/types";
+import { defineComponent, ref } from "vue";
+import MultisigParameters from "@/components/multisigParameters.vue";
+import {
+	errorMessage,
+	openErrorNotificationWithIcon,
+	tabList,
+	wrongAddress,
+	wrongAddressDes,
+} from "@/constants";
+import {
+	convertBase64ToUnit8Array,
+	formatJSON,
+	prettifyTransaction,
+} from "@/utilities";
+import { WebMode, WallectConnectSession } from "@algo-builder/web";
+import { JsonPayload } from "@algo-builder/web/build/algo-signer-types";
+import algosdk from "algosdk";
+import WalletStore from "@/store/WalletStore";
+import IconWithToolTip from "@/components/IconToolTip/IconWithToolTip.vue";
+
+let id = 0;
+
+export default defineComponent({
+	name: "Init multisignature transaction UI",
```

Comment:
name shouldn't be this long, it should be small and understandable, something like `Msig Tx Creator`